src/libraries/pkg/descriptions.json

```diff
@@ -1035,6 +1035,15 @@
       "System.Net.HttpListener"
     ]
   },
+  {
+    "Name": "System.Net.Http.Json",
+    "Description": "Provides extension methods for System.Net.Http.HttpClient and System.Net.Http.HttpContent that focus on ease of use of JSON as the data interchange format for HTTP clients (pending approval).",
```

Comment:
Cc @terrajobst to validate what should go here

Reply:
I'd say:

```suggestion
    "Description": "Provides extension methods for System.Net.Http.HttpClient and System.Net.Http.HttpContent that perform automatic serialization an deserialization using System.Text.Json.",
```